visualization/app/codeCharta/ui/edgeChooser/edgeChooser.e2e.ts

```diff
@@ -22,7 +22,6 @@ describe("MapTreeViewLevel", () => {
 	describe("EdgeChooser", () => {
 		it("should update metrics correctly after switching to a map with different metrics", async () => {
 			await fileChooser.openFiles(["./app/codeCharta/resources/sample1_with_different_edges.cc.json"])
-
```

Comment:
this commit is labeled e2e repair, you sure you didnt miss anything?

Reply:
its the whole commit so maybe you forgot to add/ push it or was it resolved somewhere else?

Reply:
Sorry that is not very descriptive ..  I think maybe I had excluded it for some reason to see why the tests were failing then I included it back again.  I have pushed every change that I have made on this branch